com.gitlab.Murmele.UDPLogger.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.Murmele.UDPLogger",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "UDPLogger",
+  "rename-desktop-file": "UDPLogger.desktop",
+  "rename-appdata-file": "UDPLogger.appdata.xml",
+  "rename-icon": "UDPLogger",
+  "finish-args": [
+    "--socket=wayland",
+	"--socket=fallback-x11",
+    "--filesystem=home",
+	"--share=network",
+	"--filesystem=xdg-config/kdeglobals:ro"],	
+    "modules": [
+		"modules/xlnt.json",
+		"modules/hdf5.json",
+		"modules/netcdf.json",
+		"modules/netcdf_c++4.json",	
```

Comment:
there is no justification to have these in separate manifest. it just make everything harder to maintain

Reply:
done